newIDE/app/src/AssetStore/AssetsList.js

```diff
@@ -130,6 +150,56 @@ const styles = {
   },
 };
 
+type PageBreakNavigationProps = {|
+  currentPage: AssetStorePageState,
+  pageBreakIndex: number,
+  setPageBreakIndex: number => void,
+  assetShortHeaders: Array<AssetShortHeader>,
+  scrollView: ?ScrollViewInterface,
+|};
+
+const PageBreakNavigation = ({
+  currentPage,
+  pageBreakIndex,
+  setPageBreakIndex,
+  assetShortHeaders,
+  scrollView,
+}: PageBreakNavigationProps) => {
+  return (
+    <Column>
+      <LineStackLayout justifyContent="center">
+        <FlatButton
+          key="previous-assets"
+          label={<Trans>Show previous assets</Trans>}
+          onClick={() => {
+            currentPage.pageBreakIndex = Math.max(
+              0,
+              (currentPage.pageBreakIndex || 0) - 1
+            );
+            setPageBreakIndex(currentPage.pageBreakIndex);
+            scrollView && scrollView.scrollToPosition(0);
+          }}
+          disabled={pageBreakIndex <= 0}
+        />
+        <RaisedButton
+          key="next-assets"
+          primary
+          label={<Trans>Show more assets</Trans>}
```

Comment:
To be consistent with "previous assets", I'm wondering if this should be "show next assets".
Though I think it's fine